src/backend/expungeservice/record_creator.py

```diff
@@ -59,23 +62,29 @@ def build_search_results(
     ) -> Tuple[List[OeciCase], List[str]]:
         errors = []
         search_results: List[OeciCase] = []
-        for alias in aliases:
-            session = requests.Session()
-            try:
-                login_response = Crawler.attempt_login(session, username, password)
-                alias_search_result = Crawler.search(
-                    session, login_response, alias.first_name, alias.last_name, alias.middle_name, alias.birth_date,
-                )
-                search_results += alias_search_result
-            except InvalidOECIUsernamePassword as e:
-                error(401, str(e))
-            except OECIUnavailable as e:
-                error(404, str(e))
-            except Exception as e:
-                errors.append(str(e))
-            finally:
-                session.close()
-        return search_results, errors
+        alias_match = RecordCreator.search_cache.__getitem__(aliases)
```

Comment:
You can use the syntax `RecordCreator.search_cache[aliases]`. See [`__getitem__`](https://docs.python.org/3/reference/datamodel.html#object.__getitem__).